scripts/deploy/invalidate-cloudfront.js

```diff
@@ -0,0 +1,37 @@
+const AWS = require('aws-sdk');
+
+const cloudfront = new AWS.CloudFront();
+const pathsToInvalidate = [
+  '/atomic/latest/*',
+  '/atomic/v1/*',
+  '/headless/latest/*',
+  '/headless/v1/*',
```

Comment:
Why are all atomic and headless `/v1` paths being invalidated?

Reply:
we invalidate the latest and "current" major, not the minors

Reply:
I see, could we programmatically set major version instead of hard-coding? Otherwise someone will need to remember to manually update the values as the major version of each project changes,